src/modules/imageresizer/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
I think this can be removed since there is already a MIT license for `PowerToys`

Reply:
@crutkas had added this License file for the original author and he had added license headers for all the existing files with this license.

Reply:
actually think this hsould be part of a NOTICE.md file

Reply:
@crutkas should the file just be renamed to NOTICE.md or is there any additional verbiage required?

Reply:
Since the [`NOTICE.md` have been added](https://github.com/microsoft/PowerToys/blob/master/NOTICE.md) removed the `LICENSE` file.